go/libraries/doltcore/sqle/dolt_index_test.go

```diff
@@ -910,7 +951,13 @@ func TestDoltIndexBetween(t *testing.T) {
 			"twopk:idx_v2v1",
 			[]interface{}{3, 4},
 			[]interface{}{4, 4},
-			[]sql.Row{{2, 2, 4, 3}, {1, 2, 3, 4}, {2, 1, 4, 4}},
+			[]sql.Row{{2, 1, 4, 4}, {2, 2, 4, 3}},
+		},
+		{
+			"twopk:idx_v2v1",
+			[]interface{}{3, 3},
+			[]interface{}{4, 4},
+			[]sql.Row{{1, 1, 3, 3}, {1, 2, 3, 4}, {2, 1, 4, 4}, {2, 2, 4, 3}},
```

Comment:
Why aren't there new tests here for mixed operators? All of these functions do this with one of the operators:

```go
testDoltIndex(t, test.keys, test.expectedRows, index, indexComp_Eq)
```

Which then does this:

```go
	for i, key := range keys {
		switch cmp {
		case indexComp_Eq:
			builder = builder.Equals(ctx, exprs[i], key)
```

So all these tests necessarily use the same operator for every key column. Seems like a pretty big gap in testing for an increase in functionality this big.

Reply:
Someone didn't read my comment lol